webapp/pages/admin/users.vue

```diff
@@ -1,15 +1,173 @@
+              <ds-button primary type="submit" icon="search" :loading="$apollo.loading" />
+            </ds-flex-item>
+          </ds-flex>
+        </ds-form>
+      </ds-card>
+    </ds-space>
+    <ds-card v-if="User && User.length">
+      <ds-table :data="User" :fields="fields" condensed>
+        <template slot="index" slot-scope="scope">
+          {{ scope.row.index }}.
+        </template>
+        <template slot="name" slot-scope="scope">
+          <nuxt-link
+            :to="{
+              name: 'profile-id-slug',
+              params: { id: scope.row.id, slug: scope.row.slug },
+            }"
+          >
+            <b>{{ scope.row.name | truncate(20) }}</b>
+          </nuxt-link>
+        </template>
+        <template slot="slug" slot-scope="scope">
+          <nuxt-link
+            :to="{
+              name: 'profile-id-slug',
+              params: { id: scope.row.id, slug: scope.row.slug },
+            }"
+          >
+            <b>{{ scope.row.slug | truncate(20) }}</b>
+          </nuxt-link>
+        </template>
+        <template slot="createdAt" slot-scope="scope">
+          {{ scope.row.createdAt | dateTime }}
+        </template>
+      </ds-table>
+      <ds-flex direction="row-reverse">
+        <ds-flex-item width="50px">
+          <ds-button @click="next" :disabled="!hasNext" icon="arrow-right" primary />
+        </ds-flex-item>
+        <ds-flex-item width="50px">
+          <ds-button @click="back" :disabled="!hasPrevious" icon="arrow-left" primary />
+        </ds-flex-item>
+      </ds-flex>
+    </ds-card>
+    <ds-card v-else>
+      <ds-placeholder>
+        {{ $t('admin.users.empty') }}
+      </ds-placeholder>
+    </ds-card>
+  </div>
 </template>
 
 <script>
-import HcEmpty from '~/components/Empty.vue'
+import gql from 'graphql-tag'
+import isemail from 'isemail'
 
 export default {
-  components: {
-    HcEmpty,
+  data() {
+    const pageSize = 15
+    return {
+      offset: 0,
+      pageSize,
+      first: pageSize,
+      User: [],
+      hasNext: false,
+      email: null,
+      filter: null,
+      form: {
+        formData: {
+          query: '',
+        },
+      },
+    }
+  },
+  computed: {
+    hasPrevious() {
+      return this.offset > 0
+    },
+    fields() {
+      return {
+        index: '#',
+        name: this.$t('admin.users.table.columns.name'),
+        slug: this.$t('admin.users.table.columns.slug'),
+        createdAt: this.$t('admin.users.table.columns.createdAt'),
+        contributionsCount: {
+          label: '🖉',
+          align: 'right',
+        },
+        commentedCount: {
+          label: '🗨',
+          align: 'right',
+        },
+        shoutedCount: {
+          label: '❤',
+          align: 'right',
+        },
+        role: {
+          label: this.$t('admin.users.table.columns.role'),
+          align: 'right',
+        },
+      }
+    },
+  },
+  apollo: {
+    User: {
+      query() {
+        return gql(`
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-07-16T06:08:58Z" title="Tuesday, July 16th 2019, 8:08:58 am +02:00">Jul 16, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,15 +1,173 @@
 <template>
-  <ds-card :header="$t('admin.users.name')">
-    <hc-empty icon="tasks" message="Coming Soon…" />
-  </ds-card>
+  <div>
+    <ds-space>
+      <ds-card :header="$t('admin.users.name')">
+        <ds-form v-model="form" @submit="submit">
+          <ds-flex gutter="small">
+            <ds-flex-item width="90%">
+              <ds-input
+                model="query"
+                :placeholder="$t('admin.users.form.placeholder')"
+                icon="search"
+              />
+            </ds-flex-item>
+            <ds-flex-item width="30px">
+              <ds-button primary type="submit" icon="search" :loading="$apollo.loading" />
+            </ds-flex-item>
+          </ds-flex>
+        </ds-form>
+      </ds-card>
+    </ds-space>
+    <ds-card v-if="User && User.length">
+      <ds-table :data="User" :fields="fields" condensed>
+        <template slot="index" slot-scope="scope">
+          {{ scope.row.index }}.
+        </template>
+        <template slot="name" slot-scope="scope">
+          <nuxt-link
+            :to="{
+              name: 'profile-id-slug',
+              params: { id: scope.row.id, slug: scope.row.slug },
+            }"
+          >
+            <b>{{ scope.row.name | truncate(20) }}</b>
+          </nuxt-link>
+        </template>
+        <template slot="slug" slot-scope="scope">
+          <nuxt-link
+            :to="{
+              name: 'profile-id-slug',
+              params: { id: scope.row.id, slug: scope.row.slug },
+            }"
+          >
+            <b>{{ scope.row.slug | truncate(20) }}</b>
+          </nuxt-link>
+        </template>
+        <template slot="createdAt" slot-scope="scope">
+          {{ scope.row.createdAt | dateTime }}
+        </template>
+      </ds-table>
+      <ds-flex direction="row-reverse">
+        <ds-flex-item width="50px">
+          <ds-button @click="next" :disabled="!hasNext" icon="arrow-right" primary />
+        </ds-flex-item>
+        <ds-flex-item width="50px">
+          <ds-button @click="back" :disabled="!hasPrevious" icon="arrow-left" primary />
+        </ds-flex-item>
+      </ds-flex>
+    </ds-card>
+    <ds-card v-else>
+      <ds-placeholder>
+        {{ $t('admin.users.empty') }}
+      </ds-placeholder>
+    </ds-card>
+  </div>
 </template>
 
 <script>
-import HcEmpty from '~/components/Empty.vue'
+import gql from 'graphql-tag'
+import isemail from 'isemail'
 
 export default {
-  components: {
-    HcEmpty,
+  data() {
+    const pageSize = 15
+    return {
+      offset: 0,
+      pageSize,
+      first: pageSize,
+      User: [],
+      hasNext: false,
+      email: null,
+      filter: null,
+      form: {
+        formData: {
+          query: '',
+        },
+      },
+    }
+  },
+  computed: {
+    hasPrevious() {
+      return this.offset > 0
+    },
+    fields() {
+      return {
+        index: '#',
+        name: this.$t('admin.users.table.columns.name'),
+        slug: this.$t('admin.users.table.columns.slug'),
+        createdAt: this.$t('admin.users.table.columns.createdAt'),
+        contributionsCount: {
+          label: '🖉',
+          align: 'right',
+        },
+        commentedCount: {
+          label: '🗨',
+          align: 'right',
+        },
+        shoutedCount: {
+          label: '❤',
+          align: 'right',
+        },
+        role: {
+          label: this.$t('admin.users.table.columns.role'),
+          align: 'right',
+        },
+      }
+    },
+  },
+  apollo: {
+    User: {
+      query() {
+        return gql(`
```


INFO: Please leave the brackets away. Just put the pure tag before the string.

In case with the brackets we have no syntax highlighting. 😢 What you can't see on your editor, probably.